documentation/book/proc-deploying-kafka-bridge-kubernetes.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge.adoc
```

Comment:
@stanlyDoge - Please can you combine the OCP and Kubernetes deployment procedures into a single procedure and apply IF statements. Please ask if you're unsure.